frontend/src/containers/Login.js

```diff
@@ -58,6 +64,7 @@ Login.propTypes = {
   splat: PropTypes.string,
   token: PropTypes.string,
   auth: PropTypes.object,
+  errorType: PropTypes.string,
```

Comment:
in this context, calling it `connectionErrorType` would be a better name,
 because else it's misleading... the reader could think it's some login error.